ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CaseEvent.java

```diff
@@ -67,7 +67,9 @@
     REVIEWED_PAPER_RESPONSE("ReviewedPaperResponse"),
     MEDIATION_FAILED("FailedMediation"),
     MEDIATION_SUCCESSFUL("SuccessfulMediation"),
-    RESET_PIN("ResetPin");
+    RESET_PIN("ResetPin"),
+    PROCEEDS_IN_CASEMAN("CaseProceedsInCaseman");
```

Comment:
The A/C say this should be just `"ProceedsInCaseman"`

Reply:
there is sub task on the story where it describes what the case event should be